examples/Internal/Sketch_Exploration/Sketch_Exploration.ino

```diff
@@ -44,7 +44,7 @@ KEYMAPS(
 )
 /* *INDENT-ON* */
 
-using namespace kaleidoscope::sketch_exploration;
+using namespace kaleidoscope::sketch_exploration; // NOLINT(build/namespaces)
```

Comment:
We _could_ remove `using namespace` from here, by adding the namespace prefixes to the things we use from `sketch_exploration`, but I feel that would make the example harder to follow, for no good reason.